packages/app/src/templates/actions/update-template-order.action.test.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { AnyAction } from "@reduxjs/toolkit"
+import thunk from "redux-thunk"
+import createMockStore from "redux-mock-store"
+import { testError } from "App/__deprecated__/renderer/store/constants"
+import { UpdateTemplateOrderError } from "App/templates/errors"
+import { Template } from "App/templates/dto"
+import { updateTemplateOrderRequest } from "App/templates/requests/update-template-order.request"
+import { updateTemplateOrder } from "App/templates/actions/update-template-order.action"
+
+const errorMock = new UpdateTemplateOrderError("Something went wrong")
+
+const mockStore = createMockStore([thunk])()
+
+const template: Template = {
+  id: "1",
+  text: "Hello world!",
+  lastUsedAt: "2",
+  order: 1,
+}
+
+beforeEach(() => {
+  mockStore.clearActions()
+})
+
+jest.mock("App/templates/requests/update-template-order.request")
+
+describe("async `updateTemplateOrder`", () => {
+  describe("when `updateTemplateOrderRequest` requests return success", () => {
+    test("fire async `updateTemplateOrder` returns updated template", async () => {
+      ;(updateTemplateOrderRequest as jest.Mock).mockReturnValue({
+        data: { ...template },
```

Comment:
Request returns the same as parameter in `updateTemplateOrder` action, so I see no point in changing `template ` here

Reply:
We should check what is returned by the `updateTemplateOrder` function. Now the test is a little ambiguous - the function receives the parameter `template` and returns the same parameter. Maybe `updateTemplateOrder` contains a mistake and returns `template` instead of `data`?

Adding such code proves that the function is correct - it returns the request result: the updated template.
```
    const updatedTemplate = {
      ...template,
      order: 123,
    }
    ;(updateTemplateOrderRequest as jest.Mock).mockReturnValue({
      data: { ...updatedTemplate },
      error: null,
    })
```